layout/js/map.js

```diff
@@ -29,6 +29,19 @@ var animate_resize_map = function(data) {
   });
 };
 
+var animate_update_tooltips = function(data) {
+  $.each(data, function() {
+    // create the tooltip text
+    var newtip = this.state_name + ': ' + this.value + ' million gallons per day'
+    // this section doesn't work. trying to assign that text to the hovertext 
+    // call for onmousemove for each object, but haven't found the way. help!
+    var newtipobj = {
+      "onmousemove": "hovertext('" + newtip + "', evt);"
+    };
+    $("#" + this.state_name + "-mouseover").mousemove = hovertext(newtip);
```

Comment:
Realizing this is a bit simpler than I thought because there is no play button - so you can't have your mouse on a state and change either the time or a category w/o moving the mouse (and thus triggering a new call to `hovertext()`)